CHANGELOG.md

```diff
@@ -12,6 +12,9 @@
   [Danielle Tomlinson](https://github.com/dantoml)
   [#421](https://github.com/CocoaPods/Xcodeproj/pull/421)
 
+* Fixed handling of xcdatamodeld packages in subfolders.
```

Comment:
Could you please add two spaces to the end of this? i.e `* Fixed handling of xcdatamodeld packages in subfolders.  `